Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/contiv/netplugin",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v74",
+	"GodepVersion": "v71",
```

Comment:
@abhinandanpb This pull request is brining in lot of unnecessary files. Basically entire ofnet/Godeps folder is being imported here, which is not required.
Two possible reasons for it:
1. you seem to be using older godep. please update to godep v74 or higher
2. you might have done godep update/save with `-r` option